samples/.env.config

```diff
@@ -0,0 +1,9 @@
+
+# Note we depend on NODE_ENV being set to dictate which of the env variables below get loaded at runtime. 
+# See README for more details.
+
+# This Admin system key
+ADMIN_SYSTEM_KEY=84abb9b30ca4ece486d4bcf7ad71
+
+# This Admin user token
+ADMIN_USER_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1aWQiOiI5NGFiYjliMzBjOTRjMGE1ZGNhOGE4ODRiYzU5Iiwic2lkIjoiMmQ5ZTAzZjUtZmQ2OC00MWM4LTg5OGYtYWQzMWE4ZjMzMmIwIiwidXQiOjIsInR0IjoxLCJleHAiOi0xLCJpYXQiOjE2NjQ1NTk5NDR9.0RrKIrs9vBY1fcp_nei3mTRhYxcZU5mdar9ribHlso0
```

Comment:
I think we should remove these example system key and tokens. If someone were to use this config as is they wouldn't get an error until getRegsitryCredentials is called and they would get an 'invalid token' error which may or may not be obvious. I'd rather provide a sample .env config which would cause the SDK to fail fast if the user doesn't supply values for system key and token

Reply:
Sure, this all is for our internal use purpose, so it's used to boost development time and not have to copy+paste every time. But will remove these values in the final push.